web/src/Routes.js

```diff
@@ -7,6 +7,7 @@ const Routes = () => {
     <Router>
```

Comment:
You should install the prettier extension ... it will automatically fix indenting issues.